duneanaobj/StandardRecord/classes_def.xml

```diff
@@ -153,6 +153,18 @@
    <version ClassVersion="10" checksum="2431680929"/>
   </class>
 
+  <class name="caf::SRNDLArID" ClassVersion="10">
+  </class>
+
+  <class name="caf::SRTMSID" ClassVersion="10">
+  </class>
+
+  <class name="caf::SRGArID" ClassVersion="10">
+  </class>
+
+  <class name="caf::SRNDLArID" ClassVersion="10">
```

Comment:
Hi Jeremy -- Looks like you put in caf::SRNDLArID twice.  My build failed with

[ 68%] Generating dictionary files for target duneanaobj_StandardRecord
Warning in <SelectionRules::CheckDuplicates>: Duplicated rule found.
Rule:
		Selected (line 156): Yes
		Attributes: 
		name = caf::SRNDLArID
		No field sel rules
		No method sel rules
Identical rule already stored:
		Selected (line 165): Yes
		Attributes: 
		name = caf::SRNDLArID
		No field sel rules
		No method sel rules

Also, it's a good idea to commit classes_def.xml after mrb has run once, as mrb fills in checksums the first time around.  I can fix these small things as it otherwise seems to build.

Reply:
Thanks Tom.  Interesting that my build didn't fail with the duplicate.  Wonder why that is.

Also, I seem to always forget to commit the updated `classes_def.xml` because I edit and use Git in an IDE on my local machine and sync it to a GPVM for building---and often I forget to sync the changes *back* from the GPVM after the checksums get filled in 🤦🏻